stories/02-features/05-Style-with-themes.stories.mdx

```diff
@@ -0,0 +1,71 @@
+<Meta title="Features|Style with Themes" />
+
+# Theming a Styled Component
+
+An assortment of minimally Styled Components is included in Irving' styled-components package. Styles are easily overwritten by providing Irving with your own custom themes. Each styled-component's configuration includes a theme property that maps to the default style included in the component package. For example, the default.js theme for the container component is:
+
+```js
+import styled from 'styled-components';
+
+/* eslint-disable import/prefer-default-export */
+export const ContainerWrapper = styled.div`
+  margin: auto;
+  width: 100%;
+`;
+/* eslint-enable */
+
+```
+
+The conditions needed to apply themes are:
+* a `themes` folder must be created in the irving directory with an `index.js` file within. It's a good practice to organize your themes by component name, so to style a container, for example, create a `container` folder within the `themes` folder.
+* a JavaScript file with your custom styling must to imported to the `index.js` file.
+* the styled-component in your template must point to your custom-theme.
+
+The styled-components's `default.js` is the best place to start when writing your own theme so copy it from `node_modules/@irvingjs/styled-components/components/<component-to-theme>/themes`. From there, add or subtract any styles you like. You can overwrite the default behavior by naming your custom theme `default.js` or keep the default behavior and add as many theme variants as you need.
+
+For example, you might want a flex container around a section, that is 90% wide with a yellow background. In order to point your app to your custom theme, you would apply the "theme" prop to your component:
```

Comment:
We shouldn't provide examples using `json` templates in here as that is a feature specific to `wp-irving`. I would say any docs that explicitly live outside the WordPress section should avoid referencing WordPress-specific functionality.

Instead, illustrate how passing a `themeName` prop to a styled component will determine which theme is rendered.